src/sympc/session/session.py

```diff
@@ -185,41 +185,98 @@ def get_protocol(self) -> Protocol:
         """
         return self.protocol
 
-    def przs_generate_random_share(
+    def _generate_random_share(
         self,
         shape: Union[tuple, torch.Size],
-    ) -> Any:
-        """Generate a random share using the two generators hold by a party.
+    ) -> List[torch.Tensor]:
+        """Generate random tensor share for the given shape.
 
         Args:
             shape (Union[tuple, torch.Size]): Shape for the share.
 
         Returns:
-            Any: ShareTensor
-
+            List[torch.Tensor] : shares generated by the generators.
         """
-        from sympc.tensor import ShareTensor
-
         gen0, gen1 = self.przs_generators
 
-        current_share = generate_random_element(
+        tensor_share1 = generate_random_element(
             tensor_type=self.tensor_type,
             generator=gen0,
             shape=shape,
         )
 
-        next_share = generate_random_element(
+        tensor_share2 = generate_random_element(
             tensor_type=self.tensor_type,
             generator=gen1,
             shape=shape,
         )
+        # We also invoke second generator for prrs as it operates in CTR mode.
```

Comment:
What is CTR?

Reply:
Sure, CTR means counter mode,given same   initial seeds to two generators,CTR enables us to generates same Pseudo Random number for subsequent invocations.This would enable parties given the same initial seeds to generate the same random number.

Reply:
> CTR enables us to generates same Pseudo Random number for subsequent invocations.This would enable parties given the same initial seeds to generate the same random number.

Could we have this in the docstring of the `PR(Z/R)S` function?